src/server/tokens-db.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+let mongodb = require('mongodb');
+let MongoClient = mongodb.MongoClient;
+let protocol = process.env.DB_PROTOCOL;
+let host = process.env.DB_HOST;
+let port = process.env.DB_PORT;
+let name = process.env.DB_NAME;
+let url = protocol+'://' + host+':' + port + '/' + name;
+function createToken(userId, userAgent) {
+  let tokenDescriptor = {};
+  let days = 7;
+  let expiresAt = new Date().getTime() + days * 24 * 60 * 60 * 1000;
+  let userAgentToString;
+  tokenDescriptor.userId = userId;
+  tokenDescriptor.userAgent = userAgent;
+  tokenDescriptor.expiresAt = expiresAt;
+  if (tokenDescriptor.userAgent.includes('Windows')) {
+    userAgentToString = 'qwerasdf';
+  } else if (tokenDescriptor.userAgent.includes('Mac')) {
+    userAgentToString = 'asdfzxcv';
+  } else if (tokenDescriptor.userAgent.includes('iPhone')) {
+    userAgentToString = 'zxcvasdf';
+  } else if (tokenDescriptor.userAgent.includes('Android')) {
+    userAgentToString = 'asdfqwer';
+  } else {
+    userAgentToString = 'zxcvzxcv';
+  }
+  tokenDescriptor.token = tokenDescriptor.userId + userAgentToString;
```

Comment:
- user agent already a string you dont need to convert it to another string
- we want to store the whole user agent, later we can create statistics from them (what kind of devices are used, what kind of systems, etc) in this way you hide a lot of important information
 - you dont create random token in this way, our site will be easily hacked. In this case, anybody can log in with an existing login if he knows the given userId. Eg i know that my friends userId is xyz and i know that he uses windows, than i can change my token to xyzqwerasdf and i can login to his account. It is not safe, it is a serious security issue. You can use a lot of modules, for example this: https://www.npmjs.com/package/randomstring